kubernetes/client/kubernetes_test.go

```diff
@@ -0,0 +1,277 @@
+// Copyright © 2018 Genome Research Limited Author: Theo Barber-Bany
```

Comment:
Author on next line. I think you've done this on all your files when you wrapped everything. You also made nonsense of many of your comments, since you didn't previously use punctuation, and now unpunctuated sentences have been joined... Please fix all as best you can...

Reply:
Ah pain, I'll read over everything in the meeting.